kitsune/users/utils.py

```diff
@@ -34,8 +34,20 @@ def _make_mail(locale):
     email_utils.send_messages([_make_mail(language_code)])
 
 
+def normalize_username(username):
+    """Removes any invalid characters from the username"""
+
+    regex = compile(r"^[\w.@+\-]+$")
```

Comment:
Should we import this directly from `django.contrib.auth.validators.UnicodeUsernameValidator.regex`?